app/lib/efile/ny/it201.rb

```diff
@@ -0,0 +1,68 @@
+module Efile
+  module Ny
+    class It201
+      # TODO: Need name ideas; could be
+      # - lines_already_filled_out
+      # - input_lines
+      # The purpose is that this stores the lines already filled out either
+      # by the client or through data imports from the IRS Form 1040.
+      attr_reader :lines
+
+      def initialize(year, filing_status, lines, it227)
+        @year = year
+
+        @filing_status = filing_status # single, mfj, that's all we support for now
+        @lines = ActiveSupport::HashWithIndifferentAccess.new(lines)
+        validate_lines
```

Comment:
not sure whether we should be doing this validation here vs expecting all values have been validated beforehand either by being part of the 1040 or validated before going into our DB